NuKeeper/Engine/RepositoryUpdater.cs

```diff
@@ -119,11 +119,11 @@ private void GitCloneToTempDir()
         {
             foreach (var current in updateSet.CurrentPackages)
             {
-                var updater = current.Path.PackageReferenceType == PackageReferenceType.ProjectFile
-                    ? (INuGetUpdater) new NuGetUpdater()
-                    : new PackagesConfigUpdater();
+                var updateCommand = current.Path.PackageReferenceType == PackageReferenceType.ProjectFile
+                    ? (IUpdatePackageCommand) new DotNetUpdatePackageCommand()
```

Comment:
This is one way to remove it - https://github.com/cohen990/NuKeeper/commit/fa708b843c1d24ff1c2365e1df477229e0b06d72

but I guess there's an argument for this hard cast being a reasonable choice given the situation